browser/components/MarkdownPreview.js

```diff
@@ -173,21 +174,29 @@ export default class MarkdownPreview extends React.Component {
     this.exportAsDocument('md')
   }
 
+  handleSaveAsHtml () {
+    this.exportAsDocument('html', function formatHtml (value) {
```

Comment:
you don't need to name function. plz just write such as `(value) => this.refs.root.contentWindow.document.documentElement.outerHTML`